LinkifiedText/index.jsx

```diff
@@ -49,6 +49,7 @@ LinkifiedText.propTypes = {
       indices: PropTypes.arrayOf(React.PropTypes.number),
     }),
   ),
+  size: PropTypes.string,
```

Comment:
Thanks for getting back to me here, @alvarezmelissa87 and @hharnisc! That feels smooth to me. I'll go ahead and push out a patch for this. 👍